packages/strapi-design-system/src/v2/SimpleMenu/SimpleMenu.js

```diff
@@ -60,33 +66,50 @@ export const MenuItem = ({ as, children, onClick, isFocused, isLink, ...props })
     }
   };
 
-  return isLink ? (
-    <OptionLink as={as} {...menuItemProps}>
-      <Box padding={2}>
-        <Typography>{children}</Typography>
-      </Box>
-    </OptionLink>
-  ) : (
-    <OptionButton onKeyDown={handleKeyDown} onMouseDown={onClick} type="button" {...menuItemProps}>
-      <Box padding={2}>
-        <Typography>{children}</Typography>
-      </Box>
-    </OptionButton>
-  );
+  if (to)
```

Comment:
I'm starting to think we won't be able to differentiate internal and external link with a prop like `to` or `href`
the all idea of this component was to remove any dependency to `react-router-dom`
with this new structure we wouldn't support routing system like Next's which we should do as described by the DS documentation:
```js

import { SimpleMenu, MenuItem } from '@strapi/design-system/SimpleMenu';
import NextLink from 'next/link';

<SimpleMenu label={Label}>
  <NextLink href="/home" passHref>
    <MenuItem>Home</MenuItem>
  </NextLink>
</SimpleMenu>;
```
_in this case would it mean that we render `OptionButton` instead of `OptionLink`?_

sorry I'm only pointing to the issue without offering a solution right now - I still need to think a little about it but I thought it would be worth sharing - maybe you or another team member would have an idea 🤔

Reply:
Ok, previously I didn't know about the `react-router-dom` removal in `v2`.
 I think the solution would be =>
1. `isExternal` should be the only prop which is deciding factor for external & internal links styling.
So this will update the `Link` component in v1 and v2 accordingly for styling.

2. For React use case, we will pass `href` and 'to' props as they are getting passed in `SimpleMenu` (my implementation)
3. For Nextjs links works outside our implementation and only styling is in our hands. So we tackled that using 1st point.

@ronronscelestes wdyt about this ?

Reply:
hi - sorry for the delay
1. for `v1` I think we can rely only on `to` and `href` prop to know which link to render since `react-router` makes it pretty straightforward to know in which context we are (if `to` we want `NavLink`, if `href` we want `Link` and pass `href` prop for the right rendering)

2. `v2` still sounds a bit tricky, I think the `isLink` prop Hicham integrated to let developer use whatever routing they want is still something we want for this component. 
Only solution I have in mind right now to keep the flexibility of external `Link` + not count on `to` to make the difference between internal and external link would be to have a `isInternal` and `isExternal` prop but I'm not sure it's a great solution 🤔 
@HichamELBSI could you give your opinion on that topic because I think support of external link in `SimpleMenu` was also something that you wanted and you're the one who implemented the `SimpleMenu` `v2` API? 👀